src/products/components/ProductUpdatePage/ProductUpdatePage.tsx

```diff
@@ -320,7 +322,32 @@ export const ProductUpdatePage: React.FC<ProductUpdatePageProps> = ({
               <Backlink href={productListUrl()}>
                 {intl.formatMessage(sectionNames.products)}
               </Backlink>
-              <PageHeader title={header}>
+              <PageHeader
+                title={header}
+                cardMenu={
+                  <>
+                    <Button
+                      data-test-id="graphiql-redirect"
+                      onClick={() => {
+                        const playgroundURL = new URL(process.env.API_URI);
+                        playgroundURL.hash = encodeGraphQLStatement({
+                          query: defaultGraphiQLQuery,
+                          headers: "",
+                          operationName: "",
+                          variables: `{ "id": "${product.id}" }`,
+                        });
+                        window.open(playgroundURL, "_blank").focus();
+                      }}
```

Comment:
It looks like potential function handler, e.g.:
```suggestion
                      onClick={openPlayground}
```
and somewhere in component:
```ts
const openPlaygroundURL = playgroundOpenHandler({
  query: defaultGraphiQLQuery,
  headers: "",
  operationName: "",
  variables: `{ "id": "${product.id}" }`,
});
```